core-plugins/src/main/java/co/cask/hydrator/plugin/batch/aggregator/AggrCommon.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.hydrator.plugin.batch.aggregator;
+
+import co.cask.cdap.api.data.schema.Schema;
+
+/**
+ * Common functions for aggregation related functionalities.
+ */
+public final class AggrCommon {
+
+  /**
+   * Checks if the type of the field is Numeric.
+   * @param fieldType type of the field to be checked.
+   * @return true if numeric, false otherwise.
+   */
+  public static boolean isNumericType(Schema.Type fieldType) {
+    if (fieldType == Schema.Type.INT || fieldType == Schema.Type.LONG ||
```

Comment:
How about
public static boolean isNumericType(Schema.Type fieldType) {
    return (fieldType == Schema.Type.INT || fieldType == Schema.Type.LONG ||  fieldType == Schema.Type.FLOAT ||    fieldType == Schema.Type.DOUBLE);
}

You can keep what you have if you think that's more readable.